gameEditor/Node.js

```diff
@@ -0,0 +1,20 @@
+export default class Node {
+    text = '';
+    num;
+    children = [];
+    parents = [];
+    constructor(text, num) {
```

Comment:
You can set the value or defaults within the constructor it self and skip declaring the class vars up top like so:

```
this.text = text || '';
this.num = num;
this.children = [];
this.parents = [];
```